source/neuropod/bindings/java/src/test/java/com/uber/neuropod/DifferentFeaturesTest.java

```diff
@@ -0,0 +1,47 @@
+/* Copyright (c) 2020 UATC, LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package com.uber.neuropod;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.*;
+
+import static org.junit.Assert.*;
+
+public class DifferentFeaturesTest {
```

Comment:
What does this mean? Can we make the name more clear?

Reply:
I wanted to test Named Dimensions. I decided to not create a separate class/test for that and decided to call it  Different Features Test where I'd put some tests that can't be organized into some domain. I guess once this class gets more tests that can be combined together, I will extract them into separate Test.

Consider that intention, any idea how I could call it or use other approach?

Reply:
I am thinking about renaming it to TesnorSpecTest that reflects a current test and I will be able to use it for all related to spec.